presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/TableLayoutHandleMatcher.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.sql.planner.assertions;
+
+import com.facebook.presto.connector.ConnectorId;
+import com.facebook.presto.metadata.TableLayoutHandle;
+import com.facebook.presto.spi.ConnectorTableLayoutHandle;
+import com.facebook.presto.testing.TestingTransactionHandle;
+
+import java.util.Optional;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+
+public final class TableLayoutHandleMatcher
+{
+    private static final TableLayoutHandle ANY = new TableLayoutHandle(new ConnectorId("any"), TestingTransactionHandle.create(), new AnyConnectorTableLayoutHandle());
```

Comment:
this is not `ANY` but some well defined `TableLayoutHandle`. To me `any` should match whatever is set, but currently matches to just `ANY` instance. Please rename it, maybe to `TestingTableLayoutHandle`.